shopinvader_assortment/models/shopinvader_backend.py

```diff
@@ -23,19 +23,17 @@ class ShopinvaderBackend(models.Model):
     def _autobind_product_from_assortment(self):
         self.ensure_one()
         product_obj = self.env["product.product"]
-        shopinvader_variant_obj = self.env["shopinvader.variant"]
         binding_wizard_obj = self.env["shopinvader.variant.binding.wizard"]
         unbinding_wizard_obj = self.env["shopinvader.variant.unbinding.wizard"]
         assortment_domain = self.product_assortment_id._get_eval_domain()
         assortment_products = product_obj.search(assortment_domain)
-        variants_binded = shopinvader_variant_obj.search(
-            [("backend_id", "=", self.id)]
+        products_bound = product_obj.search(
+            [("shopinvader_bind_ids.backend_id", "=", self.id)]
```

Comment:
do we have auto_join on backend_id field?
IMO we should and we should have an index on it if missing.

Reply:
@lmignon haha I was commenting the same :wink: 
But don't we have `auto_join=True` in v10?

Reply:
Yes, we have. We just need to decide if it is wanted to auto_join on every query ?